python_modules/libraries/dagster-airbyte/dagster_airbyte/resources.py

```diff
@@ -0,0 +1,217 @@
+import logging
+import time
+import json
+from typing import Optional
+
+import requests
+from dagster import (
+    get_dagster_logger, 
+    __version__,
+    resource,
+    StringSource,
+    Failure,
+    Field
+)
+from dagster_airbyte.types import AirbyteOutput
+from requests.exceptions import RequestException
+
+DEFAULT_POLL_INTERVAL_SECONDS = 10
+class AirbyteResource:
+    """
+    This class exposes methods on top of the Airbyte REST API.
+    """
+    RUNNING = "running"
```

Comment:
I think it would be nicer to split these constants out into a separate Enum class (maybe `AirbyteSyncState`?). That way, code that wants to interface with these values (for example, downstream ops) can reference the enum instead of a property of a resource that you won't always have access to.

Reply:
Great!